docs/src/using-plugins.md

```diff
@@ -134,8 +138,10 @@ flowchart TB
     classDef parallel fill:#FFB3B3,color:black
     classDef sequential fill:#FFD2B3,color:black
 
-    parallel(parallel):::parallel 
-    sequential(sequential):::sequential 
+    subgraph Agenda
```

Comment:
What does "Agenda" mean?

Reply:
oh, good catch! I confused it with "legend" :)